MultiSource/Benchmarks/MallocBench/cfrac/pops.c

```diff
@@ -91,10 +91,11 @@ precision palloc(size)
    register posit size;
 {
    register precision w;
-   register cacheType *kludge = pcache + size;	/* for shitty compilers */
+   register cacheType *kludge;  /* for shitty compilers */
```

Comment:
I realize you are just modifying an existing line of code w/ the comment but I don't believe this comment lines up w/ our CoC and so I think a drive by edit is called for here.

Reply:
This is just a cherry-pick of https://github.com/daanx/mimalloc-bench/commit/b517ae367c186be7365d238806b475478637d8f2

Reply:
You can put it into description or title

Reply:
I did?

Reply:
Thanks, cherry-pick make reader to expect an exact copy.